src/components/AnnotationViewer.vue

```diff
@@ -272,29 +298,69 @@ export default class AnnotationViewer extends Vue {
       strokeOpacity: 1,
       strokeWidth: 1,
     };
-    const geoJsAnnotation: IGeoJSAnnotation =
-      geojs.annotation.polygonAnnotation({ style, vertices });
+    const geoJsAnnotation = geojs.annotation.polygonAnnotation({
+      style,
+      vertices,
+    });
+    geoJsAnnotation.options("specialAnnotation", true);
 
     // Add it to the layer
-    this.samPreviewAnnotation = geoJsAnnotation;
-    this.annotationLayer.addAnnotation(this.samPreviewAnnotation);
+    this.samUnsubmittedAnnotation = geoJsAnnotation;
+    this.annotationLayer.addAnnotation(this.samUnsubmittedAnnotation);
+  }
+
+  get samLivePreviewOutput() {
+    return this.samToolState?.livePreview ?? null;
+  }
+
+  @Watch("samLivePreviewOutput")
+  onSamLivePreviewOutputChanged() {
+    // Remove previous annotation
+    if (this.samLivePreviewAnnotation) {
+      this.annotationLayer.removeAnnotation(this.samLivePreviewAnnotation);
+      this.samLivePreviewAnnotation = null;
+    }
+
+    // Create the new annotation
+    const vertices = this.samLivePreviewOutput;
+    if (!vertices) {
+      return;
+    }
+    const style = {
```

Comment:
Don't you have the possibility to customise your annotation style?

Reply:
Not yet but it could be possible in the future :D

Reply:
Don't know if it is useful for now, but maybe you can generalize this style, to make it easily customizable, what do you think?